lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/failed.yaml

```diff
@@ -283,6 +288,20 @@ states:
       delete:
         targetState: DELETE_HANDOVER_PAGE
 
+  COULD_NOT_UPDATE_DETAILS_PAGE_RFC:
+    response:
+      type: page
+      pageId: update-details-failed
+      context: repeatFraudCheck
+    events:
+      delete:
+        targetState: DELETE_HANDOVER_PAGE
+      back:
+        targetJourney: INITIAL_JOURNEY_SELECTION
```

Comment:
Is this right? The content implies that this should return you to the RP (with no identity)